debug/markers3d.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Mapbox GL JS debug page</title>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+
+    <link rel='stylesheet' href='/dist/mapbox-gl.css' />
+    <style>
+        body { margin: 0; padding: 0; }
+        html, body, #map { height: 100%; }
+
+        .mapboxgl-marker {
+            width: 50px;
+            height: 50px;
+            background: red;
+            border-radius: 25px;
+            cursor: pointer;
+        }
+    </style>
+</head>
+
+<body>
+<div id='map'></div>
+
+<script src='/dist/mapbox-gl-dev.js'></script>
+<script src='/debug/access-token-generated.js'></script>
+
+<script>
+
+var c = [-103.3684, 43.9492];
+var map = window.map = new mapboxgl.Map({
+    container: 'map',
+    zoom: 15,
+    center: c,
+    style: 'mapbox://styles/mapbox/streets-v9',
+    hash: false
+});
+
+var points = 50;
+var radius = 0.4;
+
+var p = 0;
+
+var distanceX = radius/(111.320*Math.cos(c[1]*Math.PI/180));
+var distanceY = radius/110.574;
+
+function addAnotherMarker() {
+    var theta = (p/points)*(2*Math.PI);
+    var x = distanceX*Math.cos(theta);
+    var y = distanceY*Math.sin(theta);
+
+    var toSet = [c[0]+x, c[1]+y];
+
+    p++;
+
+    var marker = new mapboxgl.Marker(null, {
+        pitchScale: true,
+        zoomScale: true,
+        nativeZoom: 15,
+        zoomFactor: 1.6,
+        offset: [-25, -50],
+        setZIndex: false
+    })
+        .setLngLat(toSet)
+        .addTo(map);
+
+    var m = marker.getElement();
+    m.onclick = function(e) {
+        alert('Hello world');
+        e.stopPropagation();
+    };
+
+    m.style.backgroundColor = 'rgb(' + [Math.floor(Math.random()*16)*16,Math.floor(Math.random()*16)*16,Math.floor(Math.random()*16)*16].join(',') + ')';
+}
+
+for (var i = 0; i < points; i++) addAnotherMarker();
+
+map.addControl(new mapboxgl.Navigation());
+
+map.on('load', function() {
+    map.setPitch(60);
+
+    var frame = function(t) {
+        map.setBearing(t/100);
+        requestAnimationFrame(frame);
+    };
+    requestAnimationFrame(frame);
+});
+</script>
+
+</body>
+</html>
```

Comment:
It makes sense that you built a debug page to use while developing this feature. 

Because repository size, simplicity, and stability is important to us, we would prefer not to merge this page into `master`.